### DISCUSSION TITLE
电脑windows10应该用什么样的配置？

### DISCUSSION THREAD

**Discussion by chinaniupai** [IrineSistiana/mosdns#671] [General]
使用别人分享的配置会无法开启，V5版本

**Comment by chinaniupai** [comment]
主要功能就是缓存和国内外分流，或者屏蔽广告这样

**Comment by kkkgo** [comment]
如果只是简单的分流你可以用作者的另一个版本：https://github.com/IrineSistiana/mosdns-cn

**Comment by chinaniupai** [comment]
> 如果只是简单的分享你可以使用作者的另一个版本：https://github.com/IrineSistiana/mosdns-cn

这个有缓存功能吗？看起来好像已经10个月没更新了

**Comment by kkkgo** [comment]
> > 如果只是简单的分享你可以使用作者的另一个版本：https://github.com/IrineSistiana/mosdns-cn
> 
> 这个有缓存功能吗？看起来好像已经10个月没更新了

有缓存功能，你可以参考项目的参数的配置。
